specification/redis/Cache.Management/PrivateEndpointConnection.tsp

```diff
@@ -0,0 +1,68 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.Cache;
+/**
+ * The Private Endpoint Connection resource.
+ */
+@parentResource(RedisResource)
+model PrivateEndpointConnection
+  is Azure.ResourceManager.CommonTypes.PrivateEndpointConnection {
+  /** The name of the private endpoint connection associated with the Azure resource */
+  @visibility(Lifecycle.Read)
+  @path
+  @segment("privateEndpointConnections")
+  @key
+  privateEndpointConnectionName: string;
+}
+
+#suppress "@azure-tools/typespec-azure-core/no-private-usage" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@Http.Private.includeInapplicableMetadataInPayload(Azure.ResourceManager.CommonTypes.PrivateEndpointConnection,
+  false
+);
+
+@armResourceOperations
+interface PrivateEndpointConnections {
+  /**
+   * Gets the specified private endpoint connection associated with the redis cache.
+   */
+  get is ArmResourceRead<
+    PrivateEndpointConnection,
+    Response = Azure.ResourceManager.CommonTypes.PrivateEndpointConnection
```

Comment:
I tried locally then find if we remove this line, SDK generation could succeed and generated openapi.json has no change, either.